src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -110,17 +80,17 @@ public boolean equals(Object other) {
         }
 
         DeleteCommand otherDeleteCommand = (DeleteCommand) other;
-        return targetIndex.equals(otherDeleteCommand.targetIndex);
+        return targetIndexes.equals(otherDeleteCommand.targetIndexes);
     }
 
     @Override
     public String toString() {
         return new ToStringBuilder(this)
-                .add("targetIndex", targetIndex)
+                .add("targetIndex", targetIndexes)
```

Comment:
Consider using `.add("targetIndex", Arrays.toString(targetIndexes))` instead.
Current implementation will add a string for the array object's reference (i.e. some giberrish text).